library/src/main/scala/scala/scalajs/js/JSConverters.scala

```diff
@@ -39,7 +39,16 @@ object JSConverters extends JSConvertersLowPrioImplicits {
 
   implicit class JSRichGenTraversableOnce[T](
       val col: GenTraversableOnce[T]) extends AnyVal {
-    @inline final def toJSArray: Array[T] = genTraversableOnce2jsArray(col)
+    final def toJSArray: Array[T] = {
```

Comment:
I think it might be worth adding the explanation why this duplicates `getTraversableOnce2jsArray` in a comment here.